test/k8sT/Updates.go

```diff
@@ -89,13 +89,8 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 			oldVersion, helpers.GetCurrentK8SEnv()))
 		return func() {}, func() {}
 	}
-	switch helpers.GetCurrentIntegration() {
-	case helpers.CIIntegrationFlannel:
-		Skip(fmt.Sprintf(
-			"Cilium %q and %q mode are not supported in K8s %q. Skipping upgrade/downgrade tests.",
-			oldVersion, helpers.CIIntegrationFlannel, helpers.GetCurrentK8SEnv()))
-		return func() {}, func() {}
-	}
+
+	SkipIfFlannel()
```

Comment:
We no longer do the following with this new function:

```
return func() {}, func () {}
```

Does `Skip` take care of that?

Reply:
Skip triggers a panic, so it's not needed at all. https://github.com/onsi/ginkgo/blob/master/ginkgo_dsl.go#L251

Reply:
TIL, that's quite inventive ;)